src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,107 @@
 package core.basesyntax;
 
+import java.util.Arrays;
```

Comment:
Avoid using utility classes like Arrays. The checklist specifies not to use Objects, Arrays, or any other util class.